en/docs/assets/js/theme.js

```diff
@@ -49,3 +49,47 @@ for (var i = 0; i < dropdowns.length; i++) {
         }
     };
 };
+
+function showDiv() {
+    document.getElementById('anonymous').style.display = "none";
+    document.getElementById('form_fields').style.display = "block";
+}
+
+function hideDiv() {
+    document.getElementById('form_fields').style.display = "none";
```

Comment:
Shall we reduce dom traversing by assign this `document.getElementById('form_fields')` to a variable and use?